.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!--
+  The description should provide all necessary information for a reviewer.
+  - What does this PR change, what's the reason for the change, how can it be tested
+-->
+&lt;Add detailed description for reviewers here.&gt;
+
+
+### Dependency release notes
+
+<!-- add links to release notes if important dependencies changed -->
+N/A
+
+### Submitter checklist
+
+- [ ] Change has been tested (on a back-end cluster)
```

Comment:
```suggestion
- [ ] Change has been tested (on a back-end cluster)
  Get system information with following commands:
  - Controllers: 'kubectl -n steward-system get deployments.apps -o yaml | grep "image:"'
    JFR: 'kubectl get clustertasks.tekton.dev steward-jenkinsfile-runner -oyaml | grep "image:"'
    HELM: 'helm -n steward-system list'
    
    
```

Reply:
This should be documented in a README and we would simply link to this. We should not describe it in the PR template.